katas/content/multi_qubit_gates/index.md

```diff
@@ -121,6 +121,82 @@ $$\alpha|00\rangle + \beta|11\rangle$$
 
 The $CNOT$ gate is self-adjoint: applying it for the second time reverses its effect.
 
+
+@[exercise]({
+    "id": "multi_qubit_gates__two_qubit_gate_1",
+    "title": "Two Qubit Gate 1",
+    "path": "./two_qubit_gate_1/",
+    "qsDependencies": [
+        "../KatasLibrary.qs"
+    ]
+})
+@[section]({
+    "id": "multi_qubit_gates__cz_gate",
+    "title": "CZ Gate"
+})
+
+## CZ Gate (Controlled-Z Gate)
```

Comment:
Nice lesson! The lesson title is in the section name, so we don't need the second header here
```suggestion
```